python/ray/autoscaler/command_runner.py

```diff
@@ -0,0 +1,155 @@
+from shlex import quote
+from typing import Any, List, Tuple, Dict, Optional
+import json
+import os
+import sys
+
+# How long to wait for a node to start, in seconds
```

Comment:
Can you please move everything else in this file, except the actual interface, either back into the private package or add a leading underscore?